app/js/components/manageApps/ManageApps.jsx

```diff
@@ -161,6 +162,27 @@ export default class ManageApps extends React.Component {
     });
   }
 
+  handleStartModules(){
+    const { appList } = this.state;
+    const allStartedModules = [];
+    appList.forEach((addon) => {
```

Comment:
Nice work. You can also achieve the same thing using `const allStartedModules = appList.filter(addon => addon.appDetails.started)`

Reply:
Cool let me update that 👍